nextcloudappstore/core/templates/app/upload.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'app/base.html' %}
+{% load i18n %}
+{% load staticfiles %}
+
+{% block head-title %}{% trans 'Upload App'%} - {% endblock %}
+
+{% block head %}
+    <script defer="defer" src="{% static 'assets/js/app-upload.js' %}"></script>
+{% endblock %}
+
+{% block apps %}
+    <h1>{% trans 'Upload App' %}</h1>
+    <section class="app-upload">
+        {% if request.user.is_authenticated %}
+        <form id="app-upload-form" method="post" action="{{ api_url }}">
```

Comment:
Good catch. I tried some different things and forgot that there. The form itself does not need that attribute, as JS does the API call using the form values, but the form is not actually submitted. (is that bad?)